sdk/xmake.lua

```diff
@@ -339,9 +460,10 @@ rule("firmware")
 			visit_all_dependencies_of(target, callback)
 		end
 
-		local boarddir, boardfile = board_file(target);
-		local board = json.loadfile(boardfile)
-
+		local boarddir, boardfile = board_file_for_target(target);
+		local board = load_board_file(json, boardfile)
+		print("Board file saved as ", target:targetfile()..".board.json")
+		json.savefile(target:targetfile()..".board.json", board)
```

Comment:
Oh thank you.  It's so obnoxious when build systems hide their computed state. :)

Reply:
It’s mostly there because I didn’t want to rewrite all of this logic for cheriot-audit (though, at least, the JSON library that it uses has JSON Patch support). This also means that we can use any JSON extensions that xmake supports and not worry because other tools can just use the output (which is plain.simple JSON).